README.md

```diff
@@ -38,6 +38,16 @@ npm install -g truffle
 npm install -g truffle-flattener
```

Comment:
I thnk we need to do 2 things here: 
1) separate the readme between setting up a development environment to develop the tool and
2) just installing and using it in a user project to audit/review the solidity code

Since this is the pkgx Hackathon, the ultimate goal would be for ceylon to be installed with pkgx. Maybe we could handle dependencies with pkgx too, so that we do noit have to require so many different things (system install, global node install...)